pkg/schema/slsa_provenance_v0.2.json

```diff
@@ -0,0 +1,180 @@
+{
+  "$id": "https://slsa.dev/provenance/v0.2",
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$defs": {
+    "DigestSet": {
+      "type": "object",
+      "propertyNames": {
+        "enum": [
+          "sha256",
+          "sha224",
+          "sha384",
+          "sha512",
+          "sha512_224",
+          "sha512_256",
+          "sha3_224",
+          "sha3_256",
+          "sha3_384",
+          "sha3_512",
+          "shake128",
+          "shake256",
+          "blake2b",
+          "blake2s",
+          "ripemd160",
+          "sm3",
+          "gost",
+          "sha1",
+          "md5"
+        ]
+      },
+      "additionalProperties": {
+        "type": "string",
+        "pattern": "^[a-f0-9]+$"
+      }
+    },
+    "Timestamp": {
+      "type": "string",
+      "format": "date-time",
+      "pattern": "Z$"
+    }
+  },
+  "type": "object",
+  "properties": {
+    "_type": {
+      "const": "https://in-toto.io/Statement/v0.1"
+    },
+    "subject": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "minLength": 1
+          },
+          "digest": {
+            "$ref": "#/$defs/DigestSet"
+          }
+        },
+        "required": [
+          "name",
+          "digest"
+        ]
+      },
+      "uniqueKeys": [
+        "/name"
+      ]
+    },
+    "predicateType": {
+      "const": "https://slsa.dev/provenance/v0.2"
+    },
+    "predicate": {
+      "type": "object",
+      "properties": {
+        "builder": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string",
+              "format": "uri"
```

Comment:
I think we need more flexibility here. This will not allow any attestations produced by Chains 0.13.0 until https://github.com/tektoncd/chains/issues/612 is resolved.

Also important to note that the current ec-cli from main also excludes attestations created by Chains 0.13.0. So this patch doesn't make things worse.

Reply:
Added [`5aad34f` (#244)](https://github.com/hacbs-contract/ec-cli/pull/244/commits/5aad34fcd11f5cdb2b262813fdd6cc221521dae4)